app/src/main/java/com/android/streetworkapp/ui/profile/Profile.kt

```diff
@@ -65,12 +75,27 @@ fun ProfileScreen(
               horizontalArrangement = Arrangement.spacedBy(10.dp, Alignment.CenterHorizontally),
               verticalAlignment = Alignment.Top) {
 
-                // profile placeholder
-                Image(
-                    painter = painterResource(id = R.drawable.profile),
-                    contentDescription = "profile picture",
-                    modifier = Modifier.size(200.dp).testTag("profilePicture"),
-                )
+                // fetch profile picture from firebase
+                val auth = Firebase.auth
+                val user = auth.currentUser
+                val photo = user?.photoUrl
```

Comment:
Isn't it is more compact to juste have it in one line like that: 
`val photo = Firebase.auth.currentUser?.photoUrl`
You don't use the intermediate variables anywhere.